invenio_app_ils/cli.py

```diff
@@ -208,6 +208,18 @@ def generate(self):
             for pid in range(1, size + 1)
         ]
 
+        demo_data_dir = os.path.join(
+            os.path.realpath("."), "invenio_app_ils", "demo_data"
```

Comment:
@kprzerwa is this an issue when deploying? I remember there was an issue with the path with vocabularies

Reply:
ok so let's test this with `cds-books` then, we keep this open.